LabExT/Tests/Utils.py

```diff
@@ -26,6 +27,13 @@ def pump_events(self):
             pass
 
 
+def mark_as_laboratory_test(klass):
+    """
+    Decorator to mark test as laboratory tests. These will be exclude, when run on CI.
+    """
+    return pytest.mark.skipif(not pytest.run_laboratory_tests, reason="skip tests that require laboratory equipment.")(klass)
```

Comment:
```suggestion
    return pytest.mark.skipif(not pytest.run_laboratory_tests, reason="skip tests that require laboratory equipment.")(class)
```

Reply:
Changed it to cls, since class is a reserved keyword.